run_qemu.sh

```diff
@@ -1148,17 +1164,30 @@ make_rootfs()
 	if [ -f ~/.vimrc ]; then
 		rsync "${rsync_opts[@]}" ~/.vim* mkosi.extra/root/
 	fi
-	mkdir -p mkosi.extra/root/bin
-	if [ -d ~/git/extra-scripts ]; then
-		rsync "${rsync_opts[@]}" ~/git/extra-scripts/bin/* mkosi.extra/root/bin/
-	fi
 	if [[ $_arg_ndctl_build == "on" ]]; then
 		if [ -n "$ndctl" ]; then
 			rsync "${rsync_opts[@]}" "$ndctl/" mkosi.extra/root/ndctl
 			prepare_ndctl_build # create mkosi.postinst which compiles
 		fi
 	fi
 
+	if [[ "$_arg_extra_dirs" ]]; then
+		extra_dirs=$(make_canonical_path $_arg_extra_dirs)
+		if [[ ! -f $extra_dirs ]]; then
+			fail "$extra_dirs not found"
+		fi
+		echo "Installing extra directories and files from: $extra_dirs"
+		for d in `cat "$extra_dirs"`; do
```

Comment:
```suggestion
		cat "$extra_dirs" | while read d; do
```
`read` is guaranteed to read line by line whereas `for ... in ...` splits on any whitespace.

Reply:
yea probably better.  cat just worked initially...  :-D